docs-source/content/faq/external-clients.md

```diff
@@ -136,21 +136,21 @@ In this example:
 
 - Additional steps are required for external clients beyond configuring the custom channel - see [Approaches](#approaches).
 
-#### Setting up a NodePort
+#### Setting up a `NodePort`
 
 ##### Getting started
 
-A Kubernetes NodePort exposes a port on each machine that hosts the Kubernetes cluster where the port is accessible from outside of a Kubernetes cluster. This port redirects network traffic to pods within the Kubernetes cluster. Setting up a Kubernetes NodePort is one approach for giving external WebLogic clients access to JMS or EJBs.
+A Kubernetes `NodePort` exposes a port on each machine that hosts the Kubernetes cluster where the port is accessible from outside of a Kubernetes cluster. This port redirects network traffic to pods within the Kubernetes cluster. Setting up a Kubernetes `NodePort` is one approach for giving external WebLogic clients access to JMS or EJBs.
```

Comment:
each machine that hosts -> each worker node in

Reply:
(they are not exposed on masters)

Reply:
Do you want this parenthetical comment, (they are not exposed on masters), added also?

Reply:
i think it is subject to configuration... either leave it out, or say `..not *normally* exposed..`